browser/net/url_context.cc

```diff
@@ -138,6 +139,8 @@ std::shared_ptr<brave::BraveRequestInfo> BraveRequestInfo::MakeCTX(
   ctx->upload_data = GetUploadData(request);
 
 #if BUILDFLAG(IPFS_ENABLED)
+  ctx->resolve_ipfs_enabled =
```

Comment:
mb it should be `resolve_ipfs_DISabled `, to avoid negation here and in the helper